go-runtime/schema/analyzers/verb.go

```diff
@@ -0,0 +1,188 @@
+package analyzers
+
+import (
+	"go/ast"
+	"go/types"
+
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/backend/schema/strcase"
+	"github.com/TBD54566975/golang-tools/go/analysis"
+	"github.com/TBD54566975/golang-tools/go/analysis/passes/inspect"
+	"github.com/TBD54566975/golang-tools/go/ast/inspector"
+	"github.com/alecthomas/types/optional"
+)
+
+// VerbExtractor extracts verbs to the module schema.
+var VerbExtractor = newExtractor("verb", extractVerbs, (*verbExtractorFact)(nil))
+
+type verbExtractorFact struct {
+	decl schema.Decl
```

Comment:
would this always be a `schema.Verb`? Or do we need to leave this as a `schema.Decl` to make the interface happy?